packages/web-service/src/pages/feedback/feedback.njk

```diff
@@ -12,6 +12,9 @@
 {% set errorMap = {
     'nps-satisfaction': {
       'any.required': { ref: '#nps-satisfaction', text: 'You have not selected an option' }
+    },
+    'withHint': {
+      'string.max': { ref: '#with-hint', text: 'You have entered too many characters' }
```

Comment:
Error message does not match the screen shot error it is missing the count

Reply:
This is for the error displayed at the top, which do not contain the char count
<img width="651" alt="image" src="https://github.com/DEFRA/wildlife-licencing/assets/141018006/c56678fc-895e-4b7b-93dd-48d3d26848f4">